core/metadata/migrations.go

```diff
@@ -239,13 +239,13 @@ func migrateSandboxes(tx *bolt.Tx) error {
 			})
 		})
 	}); merr != nil {
-		return fmt.Errorf("failed to copy sandboxes into v1 bucket: %w", err)
+		return fmt.Errorf("failed to copy sandboxes into v1 bucket: %w", merr)
 	}
 
 	for _, ns := range deletingBuckets {
 		derr := tx.DeleteBucket(ns)
 		if derr != nil {
-			return fmt.Errorf("failed to cleanup bucket %s in root: %w", ns, err)
+			return fmt.Errorf("failed to cleanup bucket %s in root: %w", ns, derr)
```

Comment:
Perhaps instead we should rename all these errors to just `err`.

Looking at this function, I see a `cerr`, a `derr` and an `merr`, and none of them seem to be special.

Using a special name not only makes it more err-prone (as evident from this PR), but also adds cognitive load.